apps/application-system/api/infra/application-system-api.ts

```diff
@@ -194,4 +194,15 @@ export const serviceSetup = (services: {
       limits: { cpu: '400m', memory: '512Mi' },
       requests: { cpu: '100m', memory: '256Mi' },
     })
-    .grantNamespaces('nginx-ingress-external', 'islandis')
```

Comment:
I would think we would need to grant both external and internal nginx ingress.

Reply:
Got confirmation from  @arnorhs that no external traffic will be to this application.